lib/elixir/src/elixir_tokenizer.erl

```diff
@@ -165,51 +168,55 @@ tokenize([$~,S,H|T] = Original, Line, Scope, Tokens) when ?is_sigil(H), ?is_upca
 
 % Char tokens
 
-tokenize([$?,$\\,$x,${,A,B,C,D,E,F,$}|T], Line, Scope, Tokens)
+tokenize([$?,$\\,$x,${,A,B,C,D,E,F,$}|T], Line, Column, Scope, Tokens)
```

Comment:
I count just 11 characters here, why are we incrementing by 12 below? :) The same for all `?\x` entries...